src/Checker/AdminChecker.php

```diff
@@ -32,10 +32,32 @@ public function isMaintenance(MaintenanceConfiguration $configuration, Request $
         /** @var string $adminPrefix */
         $adminPrefix = $this->params->get('sylius_admin.path_name');
 
+        if (str_starts_with($getRequestUri, '/_profiler') || str_starts_with($getRequestUri, '/_wdt')) {
+            return IsMaintenanceVoterInterface::ACCESS_GRANTED;
+        }
+
         if (false !== mb_strpos($getRequestUri, $adminPrefix, 1)) {
-            if ($this->requestStack->getMainRequest() === $this->requestStack->getCurrentRequest()) {
+            $session = null;
+            if (method_exists($this->requestStack, 'isMainRequest')) {
```

Comment:
This method has never existed ? I think it 's on RequestEvent instead

Reply:
I fixed it, thanks